### DISCUSSION TITLE
About PyBaMm conference (https://www.pybmc.com)

### DISCUSSION THREAD

**Discussion by alihassan90** [pybamm-team/PyBaMM#4388] [General]
Hello, Sorry if it is unrelated.

 I am planning to submit a paper for the PyBaMm conference. I have been using PyBaMm for my research and published a paper as well. I am a PhD student based outside the UK and wonder if it is possible to receive any kind of travel/accommodation funding.

**Comment by rtimms** [comment]
@brosaplanella what are the contact details for this? The website says

> Assistance
> If you are an academic student or researcher in need of financial assistance, for whatever reason, please submit a short description of why you would benefit from this, via the website. We cannot assist with travel or accommodation costs, but we hope to offer a substantial number of either partially or fully discounted tickets, depending upon need and subject to sufficient sponsorship. Preference will be given to active contributors to the PyBaMM community.

but no contact info is provided.

**Comment by valentinsulzer** [comment]
You could try speaking to Venkat Viswanathan to see if UofM has resources or grants you could apply to for this travel

**Comment by brosaplanella** [comment]
Just updated the website, sorry. The contact email is conference@pybamm.org
